src/propTypes.js

```diff
@@ -93,7 +93,26 @@ export const jobPropType = PropTypes.shape({
     false_negative: PropTypes.number,
     precision: PropTypes.number,
     recall: PropTypes.number
-  }).isRequired
+  })
+}).isRequired;
+
+export const jobRunPropType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  status: PropTypes.string.isRequired,
+  split_id: PropTypes.number.isRequired,
+  splitName: PropTypes.string,
+  type: PropTypes.oneOf([CLASSIFICATION, REGRESSION, LABELLING]).isRequired,
```

Comment:
labelling is probably not here. and what is the main difference between this and the normal Job?

Reply:
because they look similar

Reply:
Because it gave me warnings because the result in prediction and runtime are not composed by mae, rmse, etc.